.github/workflows/publish-docker-server-kotlin.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           java-version: 11
       - name: Extract branch name
         shell: bash
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF##*/})"
```

Comment:
could you please explain?

Reply:
Well, it's some black magic copied form the interwebs. The previous version took the part after `refs/heads/` and stored it in the `branch` output variable which in turn was used  as docker tag name which does not allow slash. So it failed on `feature/xxx` branch. The new version takes just the part after the last slash so it is safely slash-free.